notebooks/3_training_config.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4b6686e2",
+   "metadata": {},
+   "source": [
+    "# Anemoi training configurations\n",
+    "\n",
+    "Anemoi adopts a config-driven approach to development, and training new models makes no exception. In this notebook, we will have a brief overview of how the anemoi-training configuration system works. The configuration controls every aspect of the program, including:\n",
+    "\n",
+    "- data sources, data loading\n",
+    "- model architectures\n",
+    "- optimization hyperparameters\n",
+    "- distributed training strategy\n",
+    "- training diagnostics\n",
+    "\n",
+    "and a lot more!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88d2e271",
+   "metadata": {},
+   "source": [
+    "## Generate default configs\n",
+    "\n",
+    "Start by generating a configuration directory with"
```

Comment:
you start by generating the configuration, so why committing it to the repo in `configs/training`?

Reply:
Changed my mind halfway and forgot to remove it eheh